clients/omnitruck/dynamo_services.go

```diff
@@ -338,3 +338,12 @@ func (svc *DynamoServices) GetFilename(params *RequestParams) (string, error) {
 
 	return details.FileName, nil
 }
+
+func (svc *DynamoServices) GetPackageManagers() ([]string, error) {
+	result, err := svc.db.GetPackageManagers()
+	if err != nil {
+		svc.log.WithError(err).Error("Failed to fetch package managers from DB")
+		return nil, err
```

Comment:
need to return a fiber error so handler can parse it and use it to return correct api response